root/images/animated/placeholder.md

```diff
@@ -0,0 +1 @@
+placeholder file
```

Comment:
A Placeholder Markdown with no content is probably too confusing.
If there is anything specific to say about a sub-section, then we use a README.md
If there is nothing special and you just want to keep the folder, then a `.gitkeep` is probably what we should go for.
( I haven't  specified that one yet, but seeing this PR, I might need to update the docs.)